content/en/blog/2019/webhook/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Secure webhook management
+description: A more secure way to manage Istio webhooks.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, webhook]
+target_release: 1.4
+---
+
+Istio has two webhooks: Galley and Sidecar Injector. Both of them are crucial for Istio,
```

Comment:
Do not capitalize the sidecar injector unless you are referring specifically to the value in a config file.
```suggestion
Istio has two webhooks: Galley and the sidecar injector. Both of them are crucial for Istio,
```

Reply:
Revised accordingly.